bridges/ZoneTelechargementBridge.php

```diff
@@ -170,6 +170,6 @@ private function rewriteProtectedLink($url)
 		// Split the link using '/'
 		$parts = explode('/', $url);
 		// return the new URL using the new Link Protection system
-		return 'https://liens.onaregarde-pourvous.com/171-2/?link=' . end($parts);
+		return 'https://protecte.cam//171-2/?link=' . end($parts);
```

Comment:
Two slashes?

Reply:
Well, it seems weird, but the two slashes are present on the original website. Maybe it's a mistake on their side, but I prefer to mimic the original website :)